cluster/examples/kubernetes/ceph/common.yaml

```diff
@@ -600,29 +600,21 @@ metadata:
 rules:
 - apiGroups:
   - ""
+  - apps
```

Comment:
The RBAC changes are also needed in the [helm chart](https://github.com/rook/rook/blob/master/cluster/charts/rook-ceph/templates/clusterrole.yaml) and the [integration tests](https://github.com/rook/rook/blob/master/tests/framework/installer/ceph_manifests.go).

Reply:
Done!